### DISCUSSION TITLE
BungeeCord Bridge

### DISCUSSION THREAD

**Discussion by Ghost-chu** [QuickShop-Community/QuickShop-Hikari#536] [General]
We're developing BungeeCord bridge plugin!
For a long time, QuickShop had a big problem - unable to properly take over player chat input on BungeeCord network with Chat Plugin installed.

To solve this issue, Hikari compatibility module -- BungeeCord now available!

You will need install both QuickShop-Hikari and QuickShop BungeeCord Compaitibility Module on your Spigot/Paper and BungeeCord server!

Once you installed them and configured Spigot/Paper instance under BungeeCord mode, compatibility module will take over the chat system from BungeeCord when required.

If you are having the same problem, please try our snapshot and see if it solves your problem!

Velocity support also on the road (in plan).

**Comment by JPlayer3** [comment]
Is ChatControl Red also covered?

**Comment by Ghost-chu** [comment]
I'm not sure, try it and see!



---Original---
From: ***@***.***&gt;
Date: Fri, Nov 25, 2022 04:33 AM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [Ghost-chu/QuickShop-Hikari] BungeeCord Bridge (Discussion #536)




 
Is ChatControl Red also covered?
